### ISSUE TITLE
Unable to have Array properly stored in localstorage

### ISSUE THREAD

**Issue by malonehedges** [pinguinjkeke/vue-local-storage#5]
I have my type as `Array` and can't seem to figure out why this isn't working properly. `VueLocalStorage._properties` is somehow still `{}` and then defaults to type of `String`.

**Comment by malonehedges** [comment]
This isn't using an `Object` or `Array`, but still the `VueLocalStorage._properties` is `undefined` which then leads to assuming I want a string. Below is an example using the `vue-loader` syntax.

```
<template>
  <div id="app">
    <p>
      Welcome to your Vue.js app, {{ name }}!
    </p>
    <input v-model="inputValue" type="text" placeholder="enter ur name"/>
    <button @click="updateName()">update name</button>
  </div>
</template>

<script>
import Vue from 'vue'
import VueLocalStorage from 'vue-localstorage'

Vue.use(VueLocalStorage)

export default {
  localStorage: {
    name: {
      type: String
    }
  },
  data () {
    return {
      name: 'loading',
      inputValue: ''
    }
  },
  created () {
    this.name = this.$localStorage.get('name')
  },
  methods: {
    updateName () {
      console.log('properties is', VueLocalStorage._properties)
      this.name = this.inputValue
      this.$localStorage.set('name', this.inputValue)
    }
  }
}
</script>

<style>
html {
  height: 100%;
}

body {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100%;
  background-color: rgb(8, 26, 43);
}
</style>
```

**Comment by malonehedges** [comment]
I should note that I'm on Vue version 2.

**Comment by csscoder** [comment]
It's work for me in vue2, init this in new Vue({...}) (not component)

```
const App = new Vue({

  localStorage: {
    briefStorage: {
      type: Object,
      default: {
        hello: 'world'
      }
    }
  },
```

**Comment by pinguinjkeke** [comment]
@csscoderRU But it also should work in components.
I haven't any projects on Vue.js 2.0 to test it

**Comment by pinguinjkeke** [comment]
Fixed in v0.1.2
